web/src/lib/utils.ts

```diff
@@ -62,7 +62,7 @@ export const validateEmail = (candidate: string): boolean => {
 
 export const isSSR = () => typeof window === "undefined";
 
-export const uriHasJS = (uri: string) => /javascript:/.test(uri);
+export const uriHasJS = (uri: string) => /javascript:/i.test(uri);
```

Comment:
just found some common [examples](https://gist.github.com/xsuperbug/1aff5c1d5ddbfefb035f33dd9c8e8a72) to be used for XSS. Does it makes sense to cover them too? Seems like attacker can use them to pass that check

Reply:
@darighost, @paolodamico confirmed [here](https://tfh.slack.com/archives/C051Q5M346Q/p1702031891176449?thread_ts=1702029353.476689&cid=C051Q5M346Q), that we should handle all cases. Do you want to tackle it by yourself or should I create new PR for that cases?